homework/index.js

```diff
@@ -30,18 +30,104 @@
     return elem;
   }
 
+  function createDescription(selectedRepository, container) {
+    const descriptionContainer = createAndAppend('div', container, {
+      class: 'left-div',
+    });
+    const table = createAndAppend('table', descriptionContainer, { class: 'table' });
+    const tbody = createAndAppend('tbody', table);
+    const detailsHeader = [{ title: 'Repository', value: selectedRepository.name }];
+    const detailsBody = [
+      { title: 'Description', value: selectedRepository.description },
+      { title: 'Forks', value: selectedRepository.forks },
+      { title: 'Updated', value: new Date(selectedRepository.updated_at).toLocaleString() },
+    ];
+    detailsHeader.forEach(detail => {
+      const tr = createAndAppend('tr', tbody);
+      createAndAppend('td', tr, { text: detail.title, class: 'label' });
+      createAndAppend('a', tr, {
```

Comment:
An `a` tag cannot be used as a child of a `tr` tag. A `tr` tag can only have `td` or `th` tags as its children.